what4/src/What4/Expr/MATLAB.hs

```diff
@@ -664,71 +664,74 @@ matlabSolverReturnType f =
     CplxCosFn            -> knownRepr
     CplxTanFn            -> knownRepr
 
-ppMatlabSolverFn :: IsExpr f => MatlabSolverFn f a r -> Doc
+ppMatlabSolverFn :: IsExpr f => MatlabSolverFn f a r -> Doc ann
```

Comment:
Same thought here about an explicit annotation type (this is probably linked to/affected by the previous one anyhow).

Reply:
Same reasoning here too.